tests/foreman/api/test_reporttemplates.py

```diff
@@ -16,102 +16,56 @@
 :Upstream: No
 """
 
+from robottelo.datafactory import gen_string
 from robottelo.decorators import tier1, tier2, stubbed
 from robottelo.test import APITestCase
 
+from nailgun import entities
+
 
 class ComputeResourceTestCase(APITestCase):
     """Tests for ``katello/api/v2/report_templates``."""
 
     @tier1
-    @stubbed()
-    def test_positive_create_report(self):
-        """Create report template
+    def test_positive_e2e_crud(self):
+        """Run CRUD tests
```

Comment:
Can you please add more information in docstrings
e.g. `Execute/Run Crud operations/test for report_templates `